sdk/servicebus/service-bus/test/README.md

```diff
@@ -117,8 +117,6 @@ Populate the following variables along with the above mentioned environment vari
 AAD_CLIENT_ID=""
 AAD_CLIENT_SECRET=""
 AAD_TENANT_ID=""
-RESOURCE_GROUP=""
-AZURE_SUBSCRIPTION_ID=""
```

Comment:
Are these no longer needed?

Reply:
Yes, these were removed in a previous PR, but the readme was not updated.
https://github.com/Azure/azure-sdk-for-js/pull/6603/files#r362126519